.travis.yml

```diff
@@ -224,8 +224,8 @@ install:
   - npm install grunt-contrib-qunit
 
 script:
-  # Verify that setup.py build doesn't puke
-  - python setup.py build
+  # Test installation system-wide
+  - sudo pip install .
   # Run tests
   - WRAPT_DISABLE_EXTENSIONS=1 http_proxy=
```

Comment:
Are you sure it is enough to not cd somewhere away from the source base? I was under impression that nose would pickup datalad directory right there

Reply:
We should do that, but I wanted to go one at a time. So far it seems nothing broke with just shifting the install to an earlier step.